pkg/cmd/workspace/create/options.go

```diff
@@ -26,7 +28,7 @@ func (o *Options) Complete(args []string) error {
 }
 
 func (o *Options) Validate() error {
-	if err := util.ValidateName(o.Name); err != nil {
+	if err := util.ValidateNotDefaultName(o.Name); err != nil {
```

Comment:
If users are not allowed to create the `default` workspace with `kusion workspace create`, then how should they init a `default` workspace for a new backend? Will we provide relevant cmd?

Reply:
The default workspace will get created when you run `kusion workspace/apply/preview/destroy/build`, that is when you use workspace.